### PR TITLE
29390 django mvp

### PR DESCRIPTION
**Pull request by simon-tarr**
Created very basic models template based on the mapping-rules file we discussed earlier.

### CODE REVIEW

**Review by py5gol — COMMENTED** [review]
Good start point! There are a few files that don't go in the repo (.pyc, .sqlite, and the migrations folder). The approach is that after cloning or pulling new code, the developer runs python manage.py makemigrations and migrate to create/update the database. And python manage.py loaddata to load a fixture (we can create a fixture with python manage.py dumpdata, see https://docs.djangoproject.com/en/3.1/howto/initial-data/#providing-data-with-fixtures).

File: mapping_mvp/tables/models.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+from django.db import models
+import uuid
+
+# Create model which defines the local databases' data
+class localDB(models.Model):
```
[the remark this answers is not in the document]
Comment (simon-tarr) [reply mapping_mvp/tables/models.py:5]: Thanks, will include the style guidelines today.

File: mapping_mvp/tables/models.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    localDB_column = models.CharField(max_length = 256, unique = False)
+    localDB_value = models.CharField(max_length = 256, unique = False)
+    term = models.CharField(max_length = 256, unique = False)
+
+    def __str__(self):
+        local = f'{self.localDB_name, self.localDB_table, self.localDB_value, self.localDB_column, self.term}'
+        return local
+
+class controlledVocab(models.Model):
+    id = models.UUIDField(primary_key = True, default = uuid.uuid4, editable = False)
```
[the remark this answers is not in the document]
Comment (simon-tarr) [reply mapping_mvp/tables/models.py:17]: Complete.

File: .gitignore [hunk 3]
```diff
@@ -4,5 +4,5 @@
 *.DS_Store
 *.sqlite3
 
-*/mapping_mvp/tables/migrations/
-*/__pycache__/*
\ No newline at end of file
+mapping_mvp/tables/migrations/*
```
Comment (py5gol) [inline .gitignore:7]: **/migrations will remove the migrations folder from any future app

File: .gitignore [hunk 4]
```diff
@@ -4,5 +4,5 @@
 *.DS_Store
 *.sqlite3
```
Comment (py5gol) [inline .gitignore:6]: *.pyc

**Review by py5gol — APPROVED** [review]

**Pull request merged**